include/reelmagic.h

```diff
@@ -38,16 +38,15 @@ struct ReelMagic_VideoMixerMPEGProvider {
 
 void ReelMagic_RENDER_SetPal(uint8_t entry, uint8_t red, uint8_t green, uint8_t blue);
 
-void ReelMagic_RENDER_SetSize(const uint16_t width, const uint16_t height,
+void ReelMagic_RENDER_SetSize(const uint32_t width, const uint32_t height,
```

Comment:
You're changing the types here. Would be better to separate the type changes out into their own commit?

Reply:
Sure, but separate them out into their own commit please? They don't belong the the commit they're under now.